sdk/provisioning/Azure.Provisioning/src/Generated/Models/ManagedServiceIdentity.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+using Azure.Provisioning;
+using Azure.Provisioning.Primitives;
+using System;
+
+namespace Azure.Provisioning.Resources;
+
+/// <summary>
+/// Managed service identity (system assigned and/or user assigned identities).
+/// </summary>
+public partial class ManagedServiceIdentity : ProvisioningConstruct
```

Comment:
`ManagedServiceIdentity` is a simple model.  It has `BicepValue`s too, but it can't be named, won't appear directly in infrastructure, etc.